lib/AST/Expr.cpp

```diff
@@ -2034,18 +2037,28 @@ swift::__AbstractClosureExpr_getActorIsolation(AbstractClosureExpr *CE) {
     return (NODE)->getStartLoc();                 \
   }
 
-FORWARD_SOURCE_LOCS_TO(ClosureExpr, Body.getPointer())
+FORWARD_SOURCE_LOCS_TO(ClosureExpr, Body)
 
 Expr *ClosureExpr::getSingleExpressionBody() const {
```

Comment:
This can now potentially return `true` if a result builder produces a single expression body (which I believe it can do if the original closure is empty), however I don't think that's an issue?

Reply:
Right, the builder would have to have `buildBlock() -> ...`. Syntactic element constraint generator has `isSingleExpressionClosure` method that handles result builders as always multi-statement for its purposes and I don't think we care in other places.